src/main/java/com/iexec/common/docker/client/DockerClientInstance.java

```diff
@@ -850,4 +869,13 @@ private static DockerClient createClient(String registryAddress, String username
         }
         return dockerClient;
     }
+
+    /**
+     * Get the list of statuses where a container is considered active.
+     * 
+     * @return List.of("running", "restarting")
+     */
+    private List<String> getContainerActiveStatuses() {
+        return List.of(RESTARTING_STATUS, RUNNING_STATUS);
+    }
```

Comment:
Is there any reason to use a function instead of a constant?

Reply:
At the beginning I thought I'd need to mock it, but know I think it can be removed. Thanks for the reminder.